sites/includes/settings.liquid

```diff
@@ -12,3 +12,4 @@
 	https://wet-boew.github.io/themes-dist/GCWeb/{{ setting-packageName }}
 {%- endcapture -%}
 {%- assign setting-resourcesBasePathWetboew = "https://wet-boew.github.io/themes-dist/GCWeb/wet-boew" -%}
+{%- assign setting-root = "GCWeb/" -%}
```

Comment:
I think it should be an absolute link, otherwise it's going to be relative to the current page folder.

```suggestion
{%- assign setting-root = "/GCWeb/" -%}
```

Reply:
@duboisp I changed `"GCWeb/"` to `"/GCWeb"`. I think that would work. `"/GCWeb/"` would output a double "//".